File: data/code_pairs/fortran/2085.f90
! main.f90
program main
  implicit none
  real, dimension(3) :: vector3d
  real, dimension(3,3) :: matrix3d
  integer :: i, j

  vector3d(1) = 1.0
  vector3d(2) = 2.0
  vector3d(3) = 3.0
  matrix3d = 1.0

  ! Printing results for manual verification
  print *, 'Vector3d:'
  do i = 1, 3
     print *, vector3d(i)
  end do

  print *, 'Matrix3d:'
  do i = 1, 3
     do j = 1, 3
        print *, matrix3d(i, j)
     end do
  end do
end program main

File: data/code_pairs/cpp/2085.cpp
// main.cpp
#include <iostream>
#include <array>

int main() {
    std::array<float, 3> vector3d = {1.0f, 2.0f, 3.0f};
    std::array<std::array<float, 3>, 3> matrix3d;

    // Initialize the matrix with 1.0f
    for (auto &row : matrix3d) {
        row.fill(1.0f);
    }

    // Printing vector for manual verification
    std::cout << "Vector3d:" << std::endl;
    for (const auto &value : vector3d) {
        std::cout << value << std::endl;
    }

    // Printing matrix for manual verification
    std::cout << "Matrix3d:" << std::endl;
    for (const auto &row : matrix3d) {
        for (const auto &value : row) {
            std::cout << value << ' ';
        }
        std::cout << std::endl;
    }

    return 0;
}